02LessonTwo/in-class-project/index.html

```diff
@@ -1,10 +1,33 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>Landscape Design</title>
-  <link rel="stylesheet" href="css/style.css">
-</head>
-<body>
+  <head>
```

Comment:
We use the <head> </head> area for links to CSS and meta-data. Because of it's name, it is easily confused with the <header> section. Remember to keep all code you want seen in the browser, in the <body> of the document.